upgrade.yml

```diff
@@ -104,6 +122,16 @@
       tags: ['infra', 'percona']
   environment: "{{ env_vars|default({}) }}"
 
+# post upgrade mysql will be behind haproxy. Set endpoint
+# so db_sync commands will work until post upgrade
+- name: set mysql connection string port 3306
```

Comment:
we are setting fact to not have port 3306 in mysql connection string. Need to adjust the task name accordingly.